src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -24,16 +24,38 @@
 
 /**
  * @author Andreas Möller <am@localheinz.com>
+ * @author SpacePossum
  */
 final class NativeFunctionInvocationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /**
+     * @internal
+     */
+    const SET_ALL = '@all';
+
+    /**
+     * Subset of SET_INTERNAL.
+     *
+     * Change function call to functions known to be optimized by the Zend engine.
+     * At see: https://github.com/php/php-src/blob/master/Zend/zend_compile.c "zend_try_compile_special_func"
+     * and https://github.com/php/php-src/blob/master/ext/opcache/Optimizer/pass1_5.c
```

Comment:
please point directly to `sha`, as `master` will change in future and file can be moved
eg `https://github.com/php/php-src/blob/3a599dccb12395f9b9647dd5cd9f00322b9b644a/ext/opcache/Optimizer/pass1_5.c` (==php-7.2.6 tag)